cloudyweb/urls.py

```diff
@@ -17,9 +17,12 @@
 from rest_framework.authtoken import views as authtoken_views
 from rest_framework import routers
 from accounts import views as accounts_views
+from cloudygames import views as cloudygames_views
 
 router = routers.DefaultRouter()
 router.register(r'users', accounts_views.UserViewSet)
+router.register(r'games', cloudygames_views.GameViewSet)
+router.register(r'mygames', cloudygames_views.GameList)
```

Comment:
I believed it would be better to use `games` as the only resource for game instead of a different resource endpoint (`mygames`). which also means we should be able to handle `/games/?is_owned=1` which return game list that the user owned. 

Anyway, in this line you are registering an `APIView` instead of `ViewSet`, I am not sure why it works, have you test it out?